### PR TITLE
Add foxglove.CompressedImage support to Image panel

### PR DESCRIPTION
**Pull request by defunctzombie**
**User-Facing Changes**
Users can visualize foxglove.CompressedImage messages in the image panel.

**Description**
Fixes: #2848

### CODE REVIEW

File: packages/studio-base/src/panels/ImageView/normalizeMessage.ts [hunk 1]
```diff
@@ -97,6 +105,20 @@ export function normalizeImageMessage(
         data: typedMessage.data,
       };
     }
+    case "foxglove.CompressedImage": {
+      const typedMessage = message as FoxgloveCompressedImageMessage;
+      const sec = typedMessage.timestamp / 1000000000n;
+      const stamp = {
+        sec: Number(sec),
+        nsec: Number(typedMessage.timestamp - sec * 1000000000n),
+      };
```
Comment (jhurliman) [inline packages/studio-base/src/panels/ImageView/normalizeMessage.ts:114]: Recommend using @foxglove/rostime for this conversion which handles normalization and catches edge cases

Comment (defunctzombie) [reply packages/studio-base/src/panels/ImageView/normalizeMessage.ts:114]: What method in rostime would do this? I wasn't aware that rostime did anything with bigint. And we haven't published any rostime docs so finding the API is a hunt through the source.

Comment (defunctzombie) [reply packages/studio-base/src/panels/ImageView/normalizeMessage.ts:114]: Found this: https://github.com/foxglove/rostime/blob/main/src/timeUtils.ts#L295

Maybe we need to update the logic here. The Math.trunc is not necessary. I wonder if the `%` is worse than doing the multiply and subtract.

Comment (defunctzombie) [reply packages/studio-base/src/panels/ImageView/normalizeMessage.ts]: Updated to use fromNanoSec.

**Review by foxymiles — APPROVED** [review]

**Pull request merged**